src/ui/SetInputStep.js

```diff
@@ -51,21 +51,31 @@ function setInputStepInit() {
         video.onloadedmetadata = function(e) {
           video.play();
         };
+
+        document.getElementById('capture').addEventListener('click', function(stream){
+          context.drawImage(video, 0, 0, 400, 300);
+          options.onTakePhoto(canvas.toDataURL());
+        });
+
         document.getElementById('close').addEventListener('click', function () {
           stopStream(stream);
         });
       }
       function handleError(error) {
         console.log('navigator.getUserMedia error: ', error);
+        if(error.message == 'Permission denied' || error.message == 'NotAllowedError' || error.message == 'PermissionDismissedError'){
```

Comment:
Are there any other possible errors?

Reply:
possibly there may be more , but these are the ones which encounter after camera access denial

Reply:
Shouldn't we show the alert in case of all errors?

Reply:
Then which kind of error should we have to handle ,any suggestions?

Reply:
ok, i will add some more checks :)

Reply:
We need not handle any specific error, let us show the alert for every error.

Reply:
OK! Should we go ahead and merge this and do a follow-up, or do you want to add here? Thank you!!!

Reply:
@HarshKhandeparkar can you look at these new changes.

Reply:
Please!!